src/rollingBall.ts

```diff
@@ -23,7 +24,15 @@ import { xMean, xMaxValue, xMinValue } from 'ml-spectra-processing';
  * @param {Number} [options.windowM] - width of local window for minimization/maximization, defaults to 4% of the spectrum length
  * @param {Number} [options.windowS] - width of local window for smoothing, defaults to 8% of the spectrum length
  */
-export function rollingBall(spectrum, options = {}) {
+
+interface Options {
+  windowM: number;
+  windowS: number;
+}
+export function rollingBall(
+  spectrum: NumberArray,
+  options: Partial<Options> = {}, // not specific enough => put two optional parameters with specific type
```

Comment:
What do you mean with the comment?

Reply:
My bad. It doesn't mean anything now. just forgot to remove it.